includes/admin/tools/views/html-admin-page-system-info.php

```diff
@@ -0,0 +1,840 @@
+
+		if ( is_callable( 'gzopen' ) ) {
+			$posting['gzip']['success'] = true;
+		} else {
+			$posting['gzip']['success'] = false;
+			$posting['gzip']['note']    = sprintf( __( 'Your server does not support the %s function - this is used for file compression and decompression.', 'give' ), '<a href="https://php.net/manual/en/zlib.installation.php">gzopen</a>' );
+		}
+
+		// GD Graphics Library.
+		$posting['gd']['name'] = 'GD Graphics Library';
+		$posting['gd']['help'] = __( 'GD Graphics Library is used for dynamically manipulating images.', 'give' );
+		$posting['gd']['success'] = extension_loaded( 'gd' ) && function_exists( 'gd_info' ) ? true : false;
+
+		// Multibyte String.
+		$posting['mbstring']['name'] = 'Multibyte String';
+		$posting['mbstring']['help'] = __( 'Multibyte String (mbstring) is used to convert character encoding, like for emails or converting characters to lowercase.', 'give' );
+
+		if ( extension_loaded( 'mbstring' ) ) {
+			$posting['mbstring']['success'] = true;
+		} else {
+			$posting['mbstring']['success'] = false;
+			$posting['mbstring']['note']    = sprintf( __( 'Your server does not support the %s functions - this is required for better character encoding. Some fallbacks will be used instead for it.', 'give' ), '<a href="https://php.net/manual/en/mbstring.installation.php">mbstring</a>' );
+		}
+
+		// WP Remote Post Check.
+		$posting['wp_remote_post']['name'] = __( 'Remote Post', 'give');
+		$posting['wp_remote_post']['help'] = __( 'PayPal uses this method of communicating when sending back transaction information.', 'give' );
+
+		$response = wp_safe_remote_post( 'https://www.paypal.com/cgi-bin/webscr', array(
+			'timeout'     => 60,
+			'user-agent'  => 'Give/' . GIVE_VERSION,
+			'httpversion' => '1.1',
+			'body'        => array(
+				'cmd'     => '_notify-validate'
+			)
+		) );
+
+		if ( ! is_wp_error( $response ) && $response['response']['code'] >= 200 && $response['response']['code'] < 300 ) {
+			$posting['wp_remote_post']['success'] = true;
+		} else {
+			$posting['wp_remote_post']['note']    = __( 'wp_remote_post() failed. PayPal IPN won\'t work with your server. Contact your hosting provider.', 'give' );
+			if ( is_wp_error( $response ) ) {
+				$posting['wp_remote_post']['note'] .= ' ' . sprintf( __( 'Error: %s', 'give' ), sanitize_text_field( $response->get_error_message() ) );
+			} else {
+				$posting['wp_remote_post']['note'] .= ' ' . sprintf( __( 'Status code: %s', 'give' ), sanitize_text_field( $response['response']['code'] ) );
+			}
+			$posting['wp_remote_post']['success'] = false;
+		}
+
+		// WP Remote Get Check.
+		$posting['wp_remote_get']['name'] = __( 'Remote Get', 'give');
+		$posting['wp_remote_get']['help'] = __( 'Give plugins may use this method of communication when checking for plugin updates.', 'give' );
+
+		$response = wp_safe_remote_get( 'https://woocommerce.com/wc-api/product-key-api?request=ping&network=' . ( is_multisite() ? '1' : '0' ) );
```

Comment:
@ravinderk Yes, that should be changed to use givewp.com since the purpose of the test is to confirm that the current site can check for plugin updates.